backend/src/apiserver/main.go

```diff
@@ -89,25 +132,14 @@ func main() {
 		}
 	}
 
-	logLevel := *logLevelFlag
-	if logLevel == "" {
-		logLevel = "info"
-	}
-
-	level, err := log.ParseLevel(logLevel)
-	if err != nil {
-		log.Fatal("Invalid log level:", err)
-	}
-	log.SetLevel(level)
-
 	backgroundCtx, backgroundCancel := context.WithCancel(context.Background())
 	defer backgroundCancel()
 	wg := sync.WaitGroup{}
 	wg.Add(1)
 	go reconcileSwfCrs(resourceManager, backgroundCtx, &wg)
 	go startRpcServer(resourceManager)
```

Comment:
Since the webhook server is a separate goroutine/HTTP server, I think it makes sense to launch it here instead.